cdc/sinkv2/eventsink/txn/worker.go

```diff
@@ -61,7 +64,13 @@ func (w *worker) Close() {
 func (w *worker) runBackgroundLoop() {
 	w.wg.Add(1)
 	go func() {
-		defer w.wg.Done()
+		defer func() {
+			if err := w.backend.Close(); err != nil {
+				log.Info("transaction sink backend close fail",
+					zap.Error(err))
+			}
+			w.wg.Done()
+		}()
```

Comment:
```suggestion
                defer w.wg.Done()
		defer func() {
			if err := w.backend.Close(); err != nil {
				log.Info("transaction sink backend close fail",
					zap.Error(err))
			}
		}()
```
Avoid possible deadlocking when `w.backend.Close()` panics.